pandas/tests/internals/test_internals.py

```diff
@@ -1318,17 +1318,19 @@ def test_make_block_no_pandas_array(block_maker):
     assert result.dtype.kind in ["i", "u"]
     assert result.is_extension is False
 
-    # PandasArray, PandasDtype
-    result = block_maker(arr, slice(len(arr)), dtype=arr.dtype, ndim=arr.ndim)
-    assert result.dtype.kind in ["i", "u"]
-    assert result.is_extension is False
-
-    # ndarray, PandasDtype
-    result = block_maker(
-        arr.to_numpy(), slice(len(arr)), dtype=arr.dtype, ndim=arr.ndim
-    )
-    assert result.dtype.kind in ["i", "u"]
-    assert result.is_extension is False
+    if block_maker is make_block:
```

Comment:
i guess could have the else path here as well to assert the public api?

Reply:
this is the pseudo-public api

Reply:
right what i mean is what's the else path? (an internal path)?

Reply:
correct.  the internal path no longer takes the dtype kwd checked below